cypress/obj/loginPageObject.js

```diff
@@ -0,0 +1,59 @@
+class LoginPage{
+    visit() {
+        cy.visit('/')
+    }
+
+    getUserValidationIndicator(){
+        return cy.get('.Select.not-valid')
+    }
+
+    getLoginButton(){
+        return cy.get('[type="submit"]')
+    }
+
+    getLoginFormUserId(){
+        return cy.get('[id="loginForm.userId"]')
+    }
+
+    getUserName(userName){
+        return cy.get('[aria-label="'+ userName +'"]')
+    }
+
+    getLoginFormRoleId(){
+        return cy.get('[id="loginForm.role"]')
+    }
+
+    getUserRole(){
```

Comment:
This should also be  with parameter as for getUserName(userName) because you will need to choose another role too